libs/model/test/community/stake-transaction.spec.ts

```diff
@@ -53,55 +59,52 @@ describe.skip('Stake transactions', () => {
     await dispose()();
   });
 
-  it.skip(
-    'should create stake transactions and be able to query them',
-    async () => {
-      payload = {
-        transaction_hash:
-          '0x924f40cfea663b2579816173f048b61ab2b118e0c7c055d7b00dbd9cd15eb7c0',
-        community_id,
-      };
+  it('should create stake transactions and be able to query them', async () => {
+    payload = {
+      transaction_hash:
+        '0x924f40cfea663b2579816173f048b61ab2b118e0c7c055d7b00dbd9cd15eb7c0',
+      community_id,
+    };
 
-      let results = await command(CreateStakeTransaction(), {
-        actor,
-        payload,
-      });
-      // This address comes from the actual crypto transaction
-      expect(results?.address).to.equal(
-        '0xf6885b5aC5AE36689038dAf30184AeEB266E61f5',
-      );
-      expect(results?.stake_direction).to.equal('buy');
+    let results = await command(CreateStakeTransaction(), {
+      actor,
+      payload,
+    });
+    // This address comes from the actual crypto transaction
+    expect(results?.address).to.equal(
+      '0xf6885b5aC5AE36689038dAf30184AeEB266E61f5',
+    );
```

Comment:
Working with real services is great for fully testing the feature, but my only concern is that public RPC nodes (especially on testnet) are not always reliable and using them may make the tests flaky.

Would it make sense to mock the blockchain call or should it be a real network call?

Reply:
yes, that's why there is a TODO to create a protocol service abstraction, so these things can be mocked in unit tests... we can deal with real integrations in the next layer

Reply:
just to be clear, the ideal model library (with commands and queries) should never have a dependency to web3 or any other protocol package other than /libs/chains or any other lib with domain services (service abstractions)